docs/src/pages/layout/jss.md

```diff
@@ -0,0 +1,8 @@
+# JSS
```

Comment:
Also, I was wondering if providing a `Spacing` component like what [Airbnb is doing](https://medium.com/airbnb-engineering/rearchitecting-airbnbs-frontend-5e213efc24d2) was a useful abstraction, I finally come to the conclusion that it's not, using `theme.spacing.unit` was more flexible. That's something we could document too. But I guess it's simpler enough. cc @rosskevin